maquetado/scss/details.scss

```diff
@@ -0,0 +1,15 @@
+.text-big {
+  font-size: large;
+}
+
+.text-bold {
+  font-weight: bold;
+}
+
+.text-grey {
+  color: grey;
```

Comment:
You should have a scss file with diferent type of texts and call them: `text-1`, `text-2`, `text-3`... then you can import that file like so: `@import 'texts';`

Also, you should the same to store color variables so you can do things like:

```css
.some-class {
  color: $grey;
}
```